src/components/navigation/www/navigation.jsx

```diff
@@ -5,7 +5,7 @@ var ReactIntl = require('react-intl');
 var FormattedMessage = ReactIntl.FormattedMessage;
 var injectIntl = ReactIntl.injectIntl;
 
-var actions = require('../../../redux/actions.js');
+var actions = require('../../../redux/auth.js');
```

Comment:
For consistency elsewhere, `authActions` here? Also will make it clear which actions those are when they're used in the file, since we have different types of actions now.